src/main/java/com/ksm/wstbackoffice/mapper/AddressMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.ksm.wstbackoffice.mapper;
+
+import com.ksm.wstbackoffice.dto.AddressDto;
+import com.ksm.wstbackoffice.entity.AddressEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface AddressMapper {
+    @Mapping(source = "country.ISO3166", target = "countryISO3166")
+    AddressDto toDto(AddressEntity addressEntity);
```

Comment:
Please rename addressEntity to just entity in the mappers. addressDto to just dto, the same for country